mongodbatlas/cloud_provider_snapshot_restore_jobs.go

```diff
@@ -31,6 +31,9 @@ type CloudProviderSnapshotRestoreJobsService interface {
 	Get(context.Context, *SnapshotReqPathParameters) (*CloudProviderSnapshotRestoreJob, *Response, error)
 	Create(context.Context, *SnapshotReqPathParameters, *CloudProviderSnapshotRestoreJob) (*CloudProviderSnapshotRestoreJob, *Response, error)
 	Delete(context.Context, *SnapshotReqPathParameters) (*Response, error)
+	ServerlessList(context.Context, *SnapshotReqPathParameters, *ListOptions) (*CloudProviderSnapshotRestoreJobs, *Response, error)
+	ServerlessGet(context.Context, *SnapshotReqPathParameters) (*CloudProviderSnapshotRestoreJob, *Response, error)
+	ServerlessCreate(context.Context, *SnapshotReqPathParameters, *CloudProviderSnapshotRestoreJob) (*CloudProviderSnapshotRestoreJob, *Response, error)
```

Comment:
[nit] since go has no overloading is common to add `*With*` or `*For*` to make the distiction, this also makes it read more natural 

```suggestion
	ListForServerlessInstance(context.Context, *SnapshotReqPathParameters, *ListOptions) (*CloudProviderSnapshotRestoreJobs, *Response, error)
	GetForServerlessInstanceServerless(context.Context, *SnapshotReqPathParameters) (*CloudProviderSnapshotRestoreJob, *Response, error)
	CreateForServerlessInstance(context.Context, *SnapshotReqPathParameters, *CloudProviderSnapshotRestoreJob) (*CloudProviderSnapshotRestoreJob, *Response, error)
```

Reply:
+1

Reply:
Can you check again, pleasE?